### DISCUSSION TITLE
Limit portfolio funds for trading

### DISCUSSION THREAD

**Discussion by mertdogan** [Drakkar-Software/OctoBot#1670] [Q&A] [answered]
For example i have 1BTC but i want to use only 0.01BTC for trading. Is this possible?

**Comment by Herklos** [comment, marked as answer]
Related to https://feedback.octobot.online/posts/52/trading-fund-limitation.

Hi @mertdogan, it's not possible for now. If your post is up voted enough on feedback we'll integrate it to OctoBot roadmap.

**Comment by adamantmm** [comment]
Not built-in today, but two practical workarounds until portfolio caps land:

1. Run the bot on a sub-account / separate API key funded with only the slice you want at risk. Simplest and hardest to mess up.
2. In tentacle config, cap `trader_simulator` / order sizing so max open orders cannot exceed your budget. Less flexible but works for single-strategy setups.

For multiple strategies on one account, sub-accounts (where the exchange supports them) beat software-side caps — exchange enforces the limit even if the bot misbehaves.
